foxpass_vpn.json

```diff
@@ -3,7 +3,7 @@
     "aws_access_key":"",
     "aws_secret_key":"",
     "region":"us-west-2",
-    "source_ami":"ami-4c0bb02c"
+    "source_ami":"ami-0def3275"
```

Comment:
these changes look like spillover from PR #12.

Reply:
do we want this or should i remove the change?